travis_configure.sh

```diff
@@ -62,8 +62,9 @@ before_install()
 # Returns 0
 enable_mysqlbinlog()
 {
-    sudo sed -i -e 's/^# *server.id.*$/server_id=1/g' $1
-    sudo sed -i -e 's/^# *log.bin.*$/log_bin=mysql-bin/g' $1
+    echo '[mysqld]' | sudo bash -c "cat >> $1"
```

Comment:
yeah, as far as I understand the problem is that the command is executed as sudo but the piping to file is executed as your user.
Since this is a small command, can I suggest to improve readability to re-write it as:
```bash
sudo bash -c "echo '[mysqld]' >>  $1"
```